### PR TITLE
Fixed output of '0' in xml convert

### PR DESCRIPTION
**Pull request by shyim**
I am currently building a export of orders, but all fields where the value is 0 will be droped. This Pull Request will solve the issue

### CODE REVIEW

File: engine/Shopware/Components/Convert/Xml.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ public function _encode($array, $pos = 0, $ekey = "")
                     unset($item['_attributes'], $item['_value']);
                 }
             }
-            if (empty($item)) {
+            if (empty($item) && $item != 0) {
```
Comment (roman-1983) [inline engine/Shopware/Components/Convert/Xml.php:76]: Why don't you use `isset()` instead?

**Review by roman-1983 — CHANGES_REQUESTED** [review]
use `isset()` instead

File: engine/Shopware/Components/Convert/Xml.php (hunk 1 above)
Comment (shyim) [reply engine/Shopware/Components/Convert/Xml.php:76]: We have 2017 i can't say why.
I guess this pr will never be merged. This class is Deprecated in 5.3

**Comment by janbuecker** [comment]
We decided to close this PR because of time constraints and limited overall benefit to the product.

**Pull request closed without merging**